tests/system/python/api/test_audit.py

```diff
@@ -110,5 +148,26 @@ def test_create_audit_entry(self, fledge_url, payload, total_count):
         assert 200 == r.status
         r = r.read().decode()
         jdoc = json.loads(r)
+
+        # Storage category when with SQLite plugin, creates a child category `sqlite` hence one additional CONAD entry.
+        # CONAD, INFORMATION, '{{"name":"sqlite", ...}'
+        # so total_count is being decresed in case of 'postgres' storage engine
+        if storage_plugin == 'postgres':
+           total_count = total_count -1
+
         assert len(jdoc), "No data found"
         assert total_count == jdoc['totalCount']
+
+    @pytest.mark.parametrize("payload", [
+        ({"source": "LOGGN_X", "severity": "warning", "details": {"message": "Engine oil pressure low"}}),
+        ({"source": "LOG_X", "severity": "information", "details": {"message": "Engine oil pressure is okay."}})
+    ])
+    def test_create_nonexistent_log_code_audit_entry(self, fledge_url, payload, storage_plugin):
+        conn = http.client.HTTPConnection(fledge_url)
+        conn.request('POST', '/fledge/audit', body=json.dumps(payload))
+        r = conn.getresponse()
+        assert 400 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        print(jdoc)
```

Comment:
remove print statement

Reply:
I was thinking, we can keep such; for diagnosis using -vv such help. May be we can --verbose fixture in conftest and for system tests we can keep print for that if block?